src/App.jsx

```diff
@@ -14,33 +16,98 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const conditions = {
+  alphabetically: 'goods',
+  longer: 'length',
+  conversely: 'reverse',
+};
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function sorted(food, sortParam, other) {
+  const preparedFood = [...food];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortParam) {
+    switch (sortParam) {
+      case conditions.alphabetically:
+        preparedFood.sort((good1, good2) => good1.localeCompare(good2));
+        break;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      case conditions.longer:
+        preparedFood.sort((good1, good2) => good1.length - good2.length);
+        break;
+
+      default:
+        break;
+    }
+  }
```

Comment:
this `if` is redundant here, if we don't want to sort the array we will go to the default case and that's all